homeassistant/components/thread/config_flow.py

```diff
@@ -0,0 +1,22 @@
+"""Config flow for the Thread integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+
+class ThreadConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Thread."""
+
+    VERSION = 1
+
+    async def async_step_import(
+        self, import_data: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Set up by import from async_setup."""
+        if self._async_current_entries():
```

Comment:
Can we check this in `async_setup` to avoid instantiating all this just to abort?